shared/actions/json/profile.json

```diff
@@ -4,25 +4,26 @@
     "import * as More from '../constants/types/more'"
   ],
   "actions": {
-    "addProof": { "platform": "More.PlatformsExpandedType" },
+    "addProof": {"platform": "More.PlatformsExpandedType"},
     "backToProfile": {},
     "cancelAddProof": {},
     "cancelPgpGen": {},
     "checkProof": {},
+    "checkProof2": {"sigID": "string"},
```

Comment:
this isn't a replacement for checkproof so we should name it differently. maybe it should be like
checkBuildingProof and checkSingleProof or something